static/app/views/organizationStats/usageChart/utils.tsx

```diff
@@ -28,7 +28,9 @@ export function getDateFromMoment(
 ) {
   const days = parsePeriodToHours(interval) / 24;
   if (days >= 1) {
-    return moment.utc(m).format(FORMAT_DATETIME_DAILY);
+    return useUtc
+      ? moment.utc(m).format(FORMAT_DATETIME_DAILY)
+      : m.format(FORMAT_DATETIME_DAILY);
```

Comment:
Always moving the date value to UTC went against the documented behavior and resulted in `moment('2021-10-31')` as a parameter to come out as 'Oct 30'